### DISCUSSION TITLE
Add a opinion to set a sender name per domain.

### DISCUSSION THREAD

**Discussion by OfficialBrightLightning** [anonaddy/anonaddy#158] [Feature Requests]
Add the opinion to a sender name per domain, instead of only by account 

**Use Case** -  Say I send a email as a@github.com, and admin@github.com I would what the name for them to be GitHub, and when I send from say a@anonaddy.com, and contact@anonaddy.com I would want the sender name for them to be AnonAddy. I would not what them to both be GitHub, or AnonAddy.

Also would make since to have it on the domain page imo. 

also profile picture per domain would be nice.

**Comment by willbrowningme** [comment]
Thanks I will definitely be adding better control over the "from name" setting as soon as I can. Do you mean a profile picutre in AnonAddy on the domains page?

**Reply by OfficialBrightLightning** [reply]
Yes, to be able to set a profile picture per domain, so when you send a email from the domain the recipient would see the profile picture from any email from that domain.

**Reply by willbrowningme** [reply]
As far as I'm aware it isn't possible to set a profile picture for an email address so that everyone will see it when you send them messages.

I think you could try to use something like [http://gravatar.com/](http://gravatar.com/) or you may need to do something specific just for Gmail/Outlook.
